jbmc/unit/java_bytecode/java_replace_nondet/replace_nondet.cpp

```diff
@@ -90,6 +92,39 @@ void validate_method_removal(
   REQUIRE(replacement_nondet_exists);
 }
 
+void validate_nondets_converted(
+  std::list<goto_programt::instructiont> instructions)
+{
+  bool nondet_exists = false;
+  bool allocate_exists = false;
+  for(const auto &inst : instructions)
```

Comment:
Is it possible to use `Catch::Matchers::VectorContains` (you might need to be put instruction in a `std::vector` rather than `std::list`. It will produce nicer fail messages.

Reply:
Good suggestion - I will consider it in future, but I'd rather it didn't hold up this PR